packages/openapi-to-graphql/docs/SUBSCRIPTIONS.md

```diff
@@ -0,0 +1,279 @@
+# Subscriptions with OpenAPI-to-GraphQL
+Since version x.x.x, OpenAPI-to-GraphQL supports [GraphQL _subscription_ operations](http://spec.graphql.org/draft/#sec-Subscription). In GraphQL, using a subscription query, clients subscribe to updates on the data defined in the query. In this scneario, when data changes, the server publishes these changes to all clients that have active subscriptions for that data.
+
+The OpenAPI specification can define similar behavior using [callbacks](https://swagger.io/specification/#callbackObject): a callback defines a request that the server may initiate in response to receiving another request. Callbacks can thus be used to model publish/subscribe behavior. I.e., when the server receives a request to update some data, it can then itself issue callback requests (outside of the first request/response cycle) to any number of subscribed clients to inform about the new data.
+
+When enabling the `createSubscriptionsFromCallbacks` option, OpenAPI-to-GraphQL creates a subscription field for any operation in the given OpenAPI document that defines a `callback` object. In such cases, OpenAPI-to-GraphQL creates a [subscribe](http://spec.graphql.org/draft/#Subscribe()) function responsible to subscribing clients to receive results of callbacks being executed, and a special form of a [resolve](http://spec.graphql.org/draft/#ResolveFieldEventStream()) function, which pushes data updates to subscribed clients.
```

Comment:
I would like to add a link to the options like the following

```
[`createSubscriptionsFromCallbacks` option](https://github.com/IBM/openapi-to-graphql/tree/master/packages/openapi-to-graphql#options)
```

In addition, I think we need to add a bit more clarification. You can define multiple callbacks on an operation, multiple path items on a callback, and multiple operations on a path item. I believe in our current implementation, we are assuming (but not checking for) that there is only operation defined per path item. Even if that is the case, the wording needs to be changed because OtG would create a subscription field for all callback objects, not just one for any operation that defines callbacks as it currently suggests.